server/resources.py

```diff
@@ -65,24 +66,12 @@ def post(self):
             return responseObject, 500
 
 
-
 class UserLogin(Resource):
     def post(self):
         
         data = request.json
-        #I believe we cannot do this because the login form
-        #does not include firstName or lastName
-        #data = user_schema.load(request.json)
         current_user = UserModel.find_by_email(data['email'])
-
-        if current_user == None:
```

Comment:
This is actually needed because when u call current_user.verify_password there is a possibility that there will be no user to verify the password with and it'll throw an AttributeError